x-pack/legacy/plugins/index_management/public/sections/home/template_list/template_details/template_details.tsx

```diff
@@ -236,16 +281,58 @@ export const TemplateDetails: React.FunctionComponent<Props> = ({
 
             {templateDetails && (
               <EuiFlexItem grow={false}>
-                <EuiButtonEmpty
-                  color="danger"
-                  onClick={setTemplateToDelete.bind(null, [templateName])}
-                  data-test-subj="deleteTemplateButton"
+                {/* Manage templates context menu */}
+                <EuiPopover
+                  id="manageTemplatePanel"
+                  button={
+                    <EuiButton
+                      fill
+                      data-test-subj="manageTemplateButton"
+                      iconType="arrowDown"
+                      iconSide="right"
+                      onClick={() => setIsPopOverOpen(!isPopoverOpen)}
```

Comment:
Info: it's a recommended practice to use the callBack when updating the state based on its previous value:

```js
onClick={() => setIsPopOverOpen(prev => !prev)}
```

Reply:
good catch, fixed!